worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/util/FoliaTaskManager.java

```diff
@@ -34,23 +34,35 @@ public class FoliaTaskManager extends TaskManager {
 
     @Override
     public int repeat(@NotNull final Runnable runnable, final int interval) {
-        return fail();
+        Bukkit.getGlobalRegionScheduler().runAtFixedRate(
+                WorldEditPlugin.getInstance(),
+                scheduledTask -> runnable.run(),
+                1,
+                interval
+        );
+        return 0;
     }
 
     @Override
     public int repeatAsync(@NotNull final Runnable runnable, final int interval) {
-        backgroundExecutor.scheduleAtFixedRate(runnable, 0, ticksToMs(interval), TimeUnit.MILLISECONDS);
```

Comment:
When using the async scheduler, we need to maintain a mapping from the ScheduledTask to an id, and return that id

Reply:
I see. I will change that. I need some background for that